### PR TITLE
Show rua version

### PR DESCRIPTION
**Pull request by rowan08**
Added a footer, displaying the current version of Rua, to the base template

Trello card: https://trello.com/c/CdbtAipj/2551-add-version-footer

### CODE REVIEW

File: src/core/middleware.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from base_settings import RUA_VERSION
+
```
Comment (fradeve) [inline src/core/middleware.py:2]: Every `Class` needs to be spaced with 2 lines before and after

**Comment by rowan08** [comment]
Trello card: https://trello.com/c/CdbtAipj/2551-add-version-footer

**Review by fradeve — APPROVED** [review]

File: src/core/middleware.py (hunk 1 above)
Comment (terrasea2) [reply src/core/middleware.py:2]: Better do as he says, he's a PEP8 nazi :-1:

Comment (terrasea2) [reply src/core/middleware.py:2]: :-P

File: src/static-assets/css/over-rides.css [hunk 2]
```diff
@@ -1285,6 +1285,17 @@ body {
   background-color: #f5f5f5;
 }
 
+.footer-fixed {
+    position: fixed;
+    width: 100%;
+    bottom: 0px;
```
Comment (terrasea2) [inline src/static-assets/css/over-rides.css]: px isn't really needed if it's zero, but then it doesn't really make that much difference

**Review by terrasea2 — APPROVED** [review]

File: src/core/middleware.py (hunk 1 above)
Comment (rowan08) [reply src/core/middleware.py:2]: done

File: src/static-assets/css/over-rides.css (hunk 2 above)
Comment (rowan08) [reply src/static-assets/css/over-rides.css]: Well, I removed it in any case.

**Review by simoncrowe — APPROVED** [review]

**Pull request merged**